src/presentation/components/TopBar/TopBar.tsx

```diff
@@ -0,0 +1,30 @@
+import { useEffect, useState } from 'react';
+import { DesktopTopBar } from './DesktopTopBar';
+import { MobileTopBar } from './MobileTopBar';
+import styles from './TopBar.module.scss';
+
+export const TopBar = () => {
+	const [isMobileView, setIsMobileView] = useState(false);
+
+	useEffect(() => {
+		const setResponsiveness = () => {
+			return window.innerWidth < 950
+				? setIsMobileView(true)
+				: setIsMobileView(false);
```

Comment:
what do you think of using the condition inside of the `setIsMobileView`?

Reply:
Also if you're going to use it somewhere else you can move it to a helpers file

Reply:
Good Idea, i will do this.